pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1265,7 +1266,8 @@ private void startWorkerService(AuthenticationService authenticationService,
                 throw ioe;
             }
             LOG.info("Function worker service setup completed");
-            functionWorkerService.get().start(dlogURI, authenticationService, authorizationService);
+            // TODO figure out how to handle errors from function worker service
+            functionWorkerService.get().start(dlogURI, authenticationService, authorizationService, new ErrorNotifier());
```

Comment:
@merlimat thoughts on how we should hand this if function service is running as part of the broker?